### ISSUE TITLE
pf-codemods refuses to run: Cannot find module '@typescript-eslint/parser'

### ISSUE THREAD

**Issue by elad661** [patternfly/pf-codemods#110]
Tried running pf-codemods on my project, got the following output:

```
$ npx pf-codemods ./src
npx: installed 145 in 11.668s
Failed to load parser '@typescript-eslint/parser' declared in 'BaseConfig': Cannot find module '@typescript-eslint/parser'
Require stack:
- /home/elad/Projects/ocm/__placeholder__.js
```

**Comment by redallen** [comment]
Hey @elad661 , this is an old problem with `npx` and `eslint` that I think I've fixed in the latest version. What does `npx pf-codemods --version` say?

In the meantime, you can always install `pf-codemods` manually in your project: `npm i pf-codemods`

**Comment by elad661** [comment]
Thanks, it seems to be fixed now (`npx pf-codemods --version` says `1.2.0`)
